### DISCUSSION TITLE
Does the REST API  local UNIX socket has a default value?

### DISCUSSION THREAD

**Discussion by chenk008** [cloud-hypervisor/cloud-hypervisor#4448] [Q&A] [answered]
Does the REST API  local UNIX socket has a default value?

I have started the cloud-hypervisor , but it did not listen on any local UNIX socket.

**Comment by liuw** [comment, marked as answer]
If you don't specify `--api-socket` then no socket will be created.

You can search for `api-socket` in `src/main.rs` to see the logic there.
